source/auth/auth.rst

```diff
@@ -13,7 +13,7 @@ Driver Authentication
 :Status: Accepted
 :Type: Standards
 :Minimum Server Version: 2.6
-:Last Modified: 2020-01-13
+:Last Modified: 2020-01-16
```

Comment:
This warrants a bump to the spec version. In light of https://github.com/mongodb/specifications/pull/713#discussion_r368152348, I suppose we could just go directly to 1.9.0 with this change since the diff here is more of a behavioral change.

Reply:
I have bumped the version to 1.9.0.

Reply:
Per https://github.com/mongodb/specifications/pull/713#discussion_r369150127, please update "Spec Version" in the header a few lines up.

Reply:
Done